cloud_export_to_parquet/run_worker.py

```diff
@@ -0,0 +1,34 @@
+import asyncio
+from concurrent.futures import ThreadPoolExecutor
+
+from data_trans_activities import data_trans_and_land, get_object_keys
+from temporalio.client import Client
+from temporalio.worker import Worker
+from temporalio.worker.workflow_sandbox import (
+    SandboxedWorkflowRunner,
+    SandboxRestrictions,
+)
+from workflows import ProtoToParquet
+
+
+async def main() -> None:
+    """Main worker function."""
+    # Create client connected to server at the given address
+    client = await Client.connect("localhost:7233")
+
+    # Run the worker
+    worker: Worker = Worker(
+        client,
+        task_queue="DATA_TRANSFORMATION_TASK_QUEUE",
+        workflows=[ProtoToParquet],
+        activities=[get_object_keys, data_trans_and_land],
+        workflow_runner=SandboxedWorkflowRunner(
+            restrictions=SandboxRestrictions.default.with_passthrough_modules("boto3")
+        ),
+        activity_executor=ThreadPoolExecutor(5),
```

Comment:
```suggestion
        activity_executor=ThreadPoolExecutor(100),
```

Or should make the `max_concurrent_activities` smaller, your choice, but thread pool in theory should not be less than `max_concurrent_activities`.